DOCUMENTATION.md

```diff
@@ -122,7 +129,7 @@ You can leave out this argument if your service only has one text message sender
 
 ### Response
 
-If the request to the client is successful, the promise resolves with an `object`:
+If the request is successful, the promise resolves with a `response` `object`, such that `response.data` is:
```

Comment:
Is 'such that' a specific piece of developer language in this context, or can we say this another way?

Reply:
This is not developer specific and open to suggestions for making this easier to read